source/partials/release_notes/_release-19-7-0.md.erb

```diff
@@ -0,0 +1,69 @@
+<h4>Bug Fixes</h4>
+
+* <%= link_to_issue 6666, 'Fixes issue with GoCD server/agent not automatically starting on a server restart.' %>
+* <%= link_to_commit 'fc61beb9b34540a9068a20aecbe42326483d1e3f', 'Fixes issue with GoCD Agent DIND image which failed while running docker daemon as go user.' %>
```

Comment:
should we mention this? as we've force pushed the `19.6.0` tag, this issue never existed.

Reply:
Got it, since we force pushed this issue doesn't really exist in 19.6.0. Will remove this.